File: solutions/c#/MusicStreamingService/MusicPlayer.cs
namespace MusicStreamingService
{
    public class MusicPlayer
    {
        private Song currentSong;
        private bool isPlaying;
        private int currentTime;

        public void PlaySong(Song song)
        {
            currentSong = song;
            isPlaying = true;
            currentTime = 0;
            // Start playing the song logic
        }

        public void PauseSong()
        {
            isPlaying = false;
            // Pause logic
        }

        public void SeekTo(int time)
        {
            currentTime = time;
            // Seek logic
        }
    }
}

File: solutions/java/src/musicstreamingservice/MusicPlayer.java
package musicstreamingservice;

public class MusicPlayer {
    private Song currentSong;
    private boolean isPlaying;
    private int currentTime;

    public void playSong(Song song) {
        currentSong = song;
        isPlaying = true;
        currentTime = 0;
        // Start playing the song
        // ...
    }

    public void pauseSong() {
        isPlaying = false;
        // Pause the song
        // ...
    }

    public void seekTo(int time) {
        currentTime = time;
        // Seek to the specified time in the song
        // ...
    }
}
